app/javascript/packs/components/QuestionnaireForm/DropdownOption.jsx

```diff
@@ -27,16 +29,32 @@ class DropdownOption extends React.Component {
   }
 
   render () {
+    const color = randomColor({
```

Comment:
Color changes every time you type something - better to assign a fixed color when the option is created?